core/templates/dev/head/services/HtmlEscaperServiceSpec.ts

```diff
@@ -16,47 +16,34 @@
  * @fileoverview Unit tests for HTML serialization and escaping services.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { ErrorHandler } from '@angular/core';
+import { HtmlEscaperService } from 'services/HtmlEscaperService';
+import { LoggerService } from 'services/LoggerService';
 
-require('services/HtmlEscaperService.ts');
+describe('HTML escaper service', () => {
+  let ohe: HtmlEscaperService;
 
-describe('HTML escaper', function() {
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.upgradedServices)) {
-      $provide.value(key, value);
-    }
-  }));
-
-  describe('HTML escaper service', function() {
-    var ohe = null;
-
-    beforeEach(angular.mock.inject(function($injector) {
-      ohe = $injector.get('HtmlEscaperService');
-    }));
-
-    it('should correctly translate between escaped and unescaped strings',
-      function() {
-        var strs = ['abc', 'a&b<html>', '&&&&&'];
-        for (var i = 0; i < strs.length; i++) {
-          expect(ohe.escapedStrToUnescapedStr(
-            ohe.unescapedStrToEscapedStr(strs[i]))).toEqual(strs[i]);
-        }
-      }
-    );
+  beforeEach(() => {
+    ohe = new HtmlEscaperService(new LoggerService(new ErrorHandler()));
```

Comment:
Let's use TestBed here to avoid the nested constructor call.

`ohe = TestBed.get(HtmlEscaperService)`

Reply:
Ok Done